int32[] face_label_id
geometry_msgs/Point[] centroids
bool[] mutualGaze